### ISSUE TITLE
Colon in test description corrupts output formatting

### ISSUE THREAD

**Issue by ratiotile** [ljharb/faucet#18]
```
"tape": "^4.6.0",
"faucet": "0.0.1
npm --version 3.10.6
node --version v6.3.0
```

This line of TAP input with a colon:
`ok 200 CallableReturningEventEmitter: unregistered event does not fire listener`
produced by the tape assertion: `t.equals(testobj.hit, 'dog', "CallableReturningEventEmitter: unregistered event does not fire listener")`
results in messed-up faucet output:
![image](https://cloud.githubusercontent.com/assets/7064626/16990643/b70b0532-4e66-11e6-9b74-ba4867c6b9d3.png)
what it should look like:
![image](https://cloud.githubusercontent.com/assets/7064626/16990709/01d0091e-4e67-11e6-9900-009299cb0178.png)

**Comment by prantlf** [comment]
I didn't notice this issue at first. It appears to be the same problem as #30. A long assertion text messes up the test name printed later. I fixed it by #29.

**Comment by ljharb** [comment]
Fixed in #29.
